src/workerd/jsg/BUILD.bazel

```diff
@@ -110,24 +110,26 @@ wd_cc_library(
     ],
     # Some JSG headers can't be compiled on their own
     features = ["-parse_headers"],
-    implementation_deps = ["@simdutf"],
+    implementation_deps = [
+        ":memory-tracker",
```

Comment:
This should not be an implementation dependency – memory.h is used from header files in this target.